packages/app/src/ResizePlugin.ts

```diff
@@ -9,8 +9,12 @@ import { IApplicationOptions } from './Application';
  */
 export class ResizePlugin
 {
+    public static _resizeId: number;
```

Comment:
should `_resizeId`, `_resizeTo`, `queueResize`, and `cancelResize` be private?

`_resizeId` has `_` so it seems private
`_resizeTo` has a get/set
`queueResize`, and `cancelResize` have the `@private` tag in the docs

Reply:
Good question. I was following the lead on the access modifiers, but I guess these can be private. I think the typings for the app plugins need a little more thought.